src/common/text-area/text-area.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import type { ComponentMeta, ComponentStory } from "@storybook/react";
 import { useState } from "react";
 import { TextArea, TextAreaStatus } from "./text-area";
-import { ReactComponent as MailSVG } from "../../assets/icons/mail.svg";
-import { ReactComponent as CloseSVG } from "../../assets/icons/close.svg";
+import MailSVG from "../../assets/icons/mail.svg";
```

Comment:
can we use an absolute path here?

Reply:
Yes and no, we don't have absolute paths configured on this project and I don't think that work is in the scope of this PR.